jenkins/Jenkinsfile

```diff
@@ -28,10 +27,12 @@ pipeline {
     PROJECT = "cloudify-agent"
     VERSION = getVersion("${env.PROJECT}", "${env.BRANCH_NAME}").trim()
     PRERELEASE = getPreRelease("${env.PROJECT}", "${env.BRANCH_NAME}").trim()
-    PATH = "/root/.local/bin:$PATH"
   }
   stages{
     stage ('compatability and flake8') {
+      environment {
+        PATH = "/root/.local/bin:$PATH"
```

Comment:
Why does this need to be moved as part of the RHEL6 packaging?

Reply:
There were some changes happened recently regadring that and I cherry-picked all the related commit for the pipelines

@yuvlar any idea why you did it like this ?

Reply:
the py3compat stage needs that to be added to its PATH.
in this pipeline it uses a shared method that doesn't include this PATH, so I added it before.